File: basics/0_fib.jl
function fib(n::Int)
  n<0 && error("n must be possitive")
  n==1 && return 1
  n==0 && return 0
  fib(n-1) + fib(n-2)
end

to = parse(Int, ARGS[1])

for i in 0:to
  fib(i)
end

File: basics/0_fib.rb
def fib(n)
  raise("n must be possitive") if n < 0
  return 0 if n == 0
  return 1 if n == 1
  fib(n-1) + fib(n-2)
end

to = ARGV[0].to_i

for i in 0..to do 
  fib(i)
end